Shopify/hydrogen
# Adopt Remix v2_errorBoundary future flag

### How to test

* Go to http://localhost:3000/collections/freestylesda, you'll see `we have lost this collection`
* Go to http://localhost:3000/products/shopify-aurorasd, you'll see `we have lost this product`
* Go to http://localhost:3000/journal/our-storysda, you'll see `we have lost this page` (Generic route 404 error)
* Run project locally and throw an error anywhere, you'll see `Something’s wrong here`

#### Checklist
